### ISSUE TITLE
Build mashlib gh-pages dist automatically

### ISSUE THREAD

**Issue by michielbdejong** [solid/mashlib-dev#8]
The headers from https://solid.github.io/mashlib/dist/mashlib.js show `last-modified: Wed, 27 Nov 2019 14:16:36 GMT` 

We should have it build automatically

**Comment by michielbdejong** [comment]
Maybe using Travis?

**Comment by michielbdejong** [comment]
CC @jaxoncreed

**Comment by timbl** [comment]
Yes, I though we did.  I though @RubenVerborgh had put in a CI hook of some sort to do that a long time ago. 

This is a `mashlib` issue really not `mashlib-dev`
